egs/chime4/s5_1ch/local/rnnlm/run_lstm_back.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+# Copyright 2012  Johns Hopkins University (author: Daniel Povey)
```

Comment:
I think you should remove this backwards RNNLM stuff.  IIRC it only gives small improvements, you've implemented it in a way that breaks other scripts, and I don't see what it has to do with chime.

Reply:
This is actually important technique for us to get the performance corresponds to the 2nd best performance (without this, our result corresponds to 4th best). So, we want to keep this.

Reply:
OK but you have to implement it in a way that doesn't break the existing functionality and is clean.  Hopefully @hainan-xv can look at this and discuss with your students.